plugins/woocommerce-admin/client/marketplace/components/feedback-modal/feedback-modal.tsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Modal, Button, TextControl, TextareaControl } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { useState, useEffect } from '@wordpress/element';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import './feedback-modal.scss';
+import LikertScale from '../likert-scale/likert-scale';
```

Comment:
🗒️  **Why a fresh component?** The existing component, used by the Woo store setup checklist, would require significant modification to be suitable for use here. It's also a little hard-coded for the kinds of questions asked there/not as modular as I would like. Hence the new `<LikertScale />` component used here.